spec/models/day_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+describe Day do
+  it 'exists' do
+    day = Day.new(Time.now)
+    expect(day).to be_a(Day)
+  end
+  it "::generate_days" do
+    days = Day.generate_days(days_ago: 4, days_from_now: 7)
```

Comment:
maybe `#generate_days` for this instance method since that's how we have it throughout the rest of the app, but I'm okay either way.